imap_processing/cdf/tests/test_imap_cdf_manager.py

```diff
@@ -0,0 +1,59 @@
+# from pathlib import Path
+# from imap_processing.cdf.cdf_attribute_manager import CdfAttributeManager
+from imap_processing.cdf.imap_cdf_manager import ImapCdfAttributes
+
+
+def test_add_instrument_global_attrs():
+    # Create an ImapCdfAttributes object
+    imap_cdf_manager = ImapCdfAttributes("tests")
+    imap_cdf_manager.add_instrument_global_attrs("instrument")
+
+    # Testing data loaded in
+    imap_instrument = imap_cdf_manager.get_global_attributes("imap_test_T1_test")
+    assert imap_instrument["Data_type"] == "T1_test-one>Test-1 test one"
+    assert imap_instrument["Project"] == "STP>Solar-Terrestrial Physics"
+
+    # Testing reloading data
```

Comment:
This is already tested in the test_cdf_attribute_manager code - most of these tests are somewhat duplicating it. It's ok to have some duplicate tests, but generally, you should really try to focus in and make sure you only have one test per thing. If we changed the way we add global attrs, we would have to go and update 2 tests that test the same thing, so it's a little annoying. 

This isn't a blocker for me since this test file is pretty minimal - so I will leave it up to your judgement which tests are duplicating and which ones are adding value. Just wanted to point that out as something to think about.

Reply:
The particular reason I included both of these tests is because they are loaded in from different files. That being said though, I definitely hear what you are saying. I'll rake through my tests, and get rid of the silly ones.